ml_enabler/tests/test_upload.py

```diff
@@ -0,0 +1,61 @@
+import json
+from click.testing import CliRunner
+from ml_enabler.cli import main_group
+from ml_enabler.tests.mockserver import MockServer
+import flask
+
+
+def test_upload():
+
+    def get_model_all_response():
+        data = [
+            {
+                "modelId": 1,
+                "created": "2019-05-31T10:55:27.636457",
+                "name": "looking_glass",
+                "source": "developmentseed",
+                "dockerhubUrl": "dockerhubUrl"
+            }
+        ]
+
+        res = flask.Response(
+              json.dumps(data),
+              mimetype='application/json'
+        )
+        res.headers['Content-Length'] = len(res.get_data())
```

Comment:
@geohacker let's remove this bit of magic .. this was trying to solve for random failures from the mock server when fetching predictions -- we shouldn't need this - in case we start seeing random failures while running tests, we can consider adding back.